src/components/build_meta.js

```diff
@@ -12,6 +13,7 @@ class BuildMeta extends React.Component {
         <em>{build.author}</em>
         <span>pushed to</span>
         <em>{build.branch}</em>
+        <Timestamp time={build.created_at || build.enqueued_at} format="full" className="time"/>
```

Comment:
Thought it'd be a good idea to show more detailed time information rather than just `X days ago`. But it also makes sense to keep time representation consistent across the board. Will change this to use `react-timeago`.